src/modules/linagora.esn.user.status/frontend/app/app.js

```diff
@@ -1,8 +1,17 @@
-(function() {
+(function(angular) {
   'use strict';
 
-  angular.module('linagora.esn.user-status', [
-    'restangular',
-    'esn.lodash-wrapper'
-  ]);
-})();
+  angular.module('linagora.esn.user-status', ['esn.websocket', 'esn.session']);
+})(angular);
+
+require('../../../../frontend/js/modules/websocket.js');
+require('../../../../frontend/js/modules/session.js');
```

Comment:
Not for this PR, but we will have to use webpack module aliasing to avoid `../../../../../../..//../..//.//..///.....//.../.` :p

Reply:
right after splitting this module?

Reply:
No, we will do it later